package.json

```diff
@@ -33,6 +33,9 @@
     "dist",
     "dist-es5-module"
   ],
+  "engines": {
+    "node": "7.10.0"
```

Comment:
Sorry I mislead you in adding this line, it should not be here because it will then produce unnecessary warnings when installed: https://docs.npmjs.com/files/package.json#engines

You can safely remove it. The only time this is useful is when you want your dev environment (for devs of the lib), to be tied to a specific nodejs version (this is useful)